docs/forms/translations/ja/respondent-email-notifications.md

```
@@ -0,0 +1,94 @@
+keywords:
+  - grida
+  - forms
+  - email notifications
+  - respondent email
+  - ciam
+format: md
+doc_tasks:
+  - update
+---
+
+### 回答者向けメール通知
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)